scripts/release-notes.py

```diff
@@ -88,17 +109,57 @@ def main(token, repo, num_commits, exclude_dependabot):
         if not pulls:
             short_sha = sha[:7]
             commit_url = commit['html_url']
-            print(f'* {msg} ([@{author}]({author_url}) in [{short_sha}]({commit_url}))')
+            
+            result = f'* {msg} ([@{author}]({author_url}) in [{short_sha}]({commit_url}))'
+            other_results.append(result)
             continue
 
         pull = pulls[0]
         pull_id = pull['number']
         pull_url = pull['html_url']
         msg = msg.replace(f'(#{pull_id})', '').strip()
-        print(f'* {msg} ([@{author}]({author_url}) in [#{pull_id}]({pull_url}))')
+
+        # Check if the pull request has changelog label
+        pull_labels = get_pull_request_labels(token, args.repo, pull_id)
+        changelog_labels = [label for label in pull_labels if label.startswith('changelog:')]
+
+        category = UNCATTEGORIZED
+        if changelog_labels:
+            for cat in categories:
+                if changelog_labels[0].startswith(cat['label']):
+                    category = cat['title']
+                    break
+            
+        result = f'* {msg} ([@{author}]({author_url}) in [#{pull_id}]({pull_url}))'
+        if category == UNCATTEGORIZED:
+            other_results.append(result)
+        else:
+            category_results[category].append(result)
+
+    # Print categorized pull requests
+    for category, results in category_results.items():
+        if results and category:
+            print(f'{category}:')
```

Comment:
new line after header

Reply:
Do we need an empty line after the header? Currently it is like:

Header 1:
PRs Table

Header 2:
PRs Table